qiime2/core/testing/transformer.py

```diff
@@ -94,6 +97,18 @@ def _1000(ff: IntSequenceFormat) -> IntSequenceFormatV2:
     return new_ff
 
 
+@dummy_plugin.register_transformer
+def _1100(df: IntSequenceMultiFileDirectoryFormat) -> \
+        IntSequenceDirectoryFormat:
+    return IntSequenceDirectoryFormat()
```

Comment:
The `IntSequenceMultiFileDirectoryFormat` doesn't actually contain any data :laughing:. The entire body of the class is just `pass`. Do we want it to have data in it? I just went with the path of least resistance when creating it because data wasn't needed to test this behavior.

Reply:
fair enough, let's ignore then, but add a comment indicating this is for testing `get_formats` only